streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -424,11 +424,11 @@ class CheckpointSuite extends TestSuiteBase {
             }
           }
         }
-        clock.advance(batchDuration.milliseconds)
         eventually(eventuallyTimeout) {
           // Wait until all files have been recorded and all batches have started
           assert(recordedFiles(ssc) === Seq(1, 2, 3) && batchCounter.getNumStartedBatches === 3)
         }
+        clock.advance(batchDuration.milliseconds)
```

Comment:
This line will add one batch, so I move it here. And because of this line, the clock of the recovery StreamingContext will be 14000ms.